proto/axelar/tss/v1beta1/tx.proto

```diff
@@ -90,11 +90,12 @@ message HeartBeatRequest {
   bytes sender = 1 [ (gogoproto.casttype) =
                          "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
 
+  // Deprecated: this field will be removed in the next release
```

Comment:
```suggestion
```
we can't actually remove the proto or the field due to avoid breaking rosetta i believe. we'll just remove the handler later when we refactor this

Reply:
why can't we reserve the number and remove it?

Reply:
ah, yes, forgot about reserving